src/App.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+
+it('renders without crashing', () => {
```

Comment:
Ooh, let's add this test to the travis config so we make sure that we don't push a breaking change 🎉

Reply:
oh, think it already runs automatically because this is run via `npm test` ~ cool!